ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -136,7 +135,11 @@ public void sendRequest(
     requestBuilder.headers(requestHeaders);
 
     if (data == null) {
-      requestBuilder.method(method, null);
+      if (method.equals("POST") || method.equals("PUT") || method.equals("PATCH")){
```

Comment:
Can you put this in a method, like `getEmptyBody(String method)` and put it in `RequestBodyUtil`. Then you'd just have:
    requestBuilder.method(method, getEmptyBody(method));